exer1-intf-table.py

```diff
@@ -1,17 +1,18 @@
 import json
 
-jsondata = open('exer1-interface-data.json').read()
+with open('exer1-interface-data.json', 'r') as jsondata:
```

Comment:
https://docs.python.org/2/tutorial/inputoutput.html

using a with loop with files is good practice